.cfnlintrc

```diff
@@ -0,0 +1,15 @@
+templates:
+  - templates/*
+  
+# Used by cfn-lint GitHub Action
+ignore_checks:
+  # W4002: As the resource "metadata" section contains reference to a "NoEcho" parameter DBMasterUserPassword, CloudFormation will display the parameter value in plaintext
+  - W4002
+  # E3012: Property Resources/EFSCname/Properties/TTL should be of type Long
+  - E3012
+  # E1001: Top level template section tests is not valid
+  - E1001
+  # W3011: Both UpdateReplacePolicy and DeletionPolicy are needed to protect Resources/EncryptionKey from deletion
+  - W3011
+  # E3029: You cannot specify CopyTagsToSnapshot for Aurora AWS::RDS::DBInstance at Resources/AuroraDB3/Properties/CopyTagsToSnapshot
+  - E3029
```

Comment:
[debating whether we should remove this property from the rule itself](https://github.com/aws-cloudformation/cfn-python-lint/pull/1409)